FWCore/Framework/src/EventPrincipal.cc

```diff
@@ -214,7 +213,7 @@ namespace edm {
                                  std::optional<ProductProvenance> productProvenance) const {
     assert(!bd.produced());
     if (productProvenance) {
-      productProvenanceRetrieverPtr()->insertIntoSet(std::move(*productProvenance));
+      productProvenanceRetrieverPtr()->insertIntoSet(*productProvenance);
```

Comment:
Just to note for future reference, `ProductProvenance` is as efficient to copy as it is to move.